### PR TITLE
Add CloudBeaver sample config

### PR DESCRIPTION
**Pull request by ofersadan85**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

### CODE REVIEW

**Comment by Roxedus** [comment]
Please fix the line endings

**Comment by ofersadan85** [comment]
@Roxedus done, sorry about that

File: cloudbeaver.subdomain.conf.sample [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        # enable the next two lines for ldap auth
+        #auth_request /auth;
+        #error_page 401 =200 /ldaplogin;
+
+        # enable for Authelia
+        #include /config/nginx/authelia-location.conf;
+
+        include /config/nginx/proxy.conf;
+        include /config/nginx/resolver.conf;
+
```
Comment (Roxedus) [inline cloudbeaver.subdomain.conf.sample:37]: Please keep this line consistent with the template

**Review by Roxedus — CHANGES_REQUESTED** [review]

File: cloudbeaver.subdomain.conf.sample (hunk 1 above)
Comment (ofersadan85) [reply cloudbeaver.subdomain.conf.sample]: done, hopefully

**Pull request merged**